tensorflow_addons/layers/stochastic_depth.py

```diff
@@ -0,0 +1,90 @@
+import tensorflow as tf
+from typeguard import typechecked
+
+
+@tf.keras.utils.register_keras_serializable(package="Addons")
+class StochasticDepth(tf.keras.layers.Layer):
+    r"""Stochastic Depth layer.
+
+    Implements Stochastic Depth as described in
+    [Deep Networks with Stochastic Depth](https://arxiv.org/abs/1603.09382), to randomly drop residual branches
+    in residual architectures.
+
+    Usage:
+    Residual architectures with fixed depth, use residual branches that are merged back into the main network
+    by adding the residual branch back to the input:
+
+    ```python
+    residual = tf.keras.layers.Conv2D(...)(input)
+
+    return tf.keras.layers.Add()([input, residual])
+    ```
+
+    StochasticDepth acts as a drop-in replacement for the addition:
+
+    ```python
+    residual = tf.keras.layers.Conv2D(...)(input)
+
+    return tfa.layers.StochasticDepth()([input, residual])
+    ```
+
+    At train time, StochasticDepth returns:
+
+    ```python
+    x[0] + b_l * x[1]
+    ```
+
+    , where b_l is a random Bernoulli variable with probability p(b_l == 1) == p_l
+
+    At test time, StochasticDepth rescales the activations of the residual branch based on the survival probability:
+
+    ```python
+    x[0] + p_l * x[1]
+    ```
```

Comment:
I guess you want to use mathjax expression here and above, you can wrap your math expr with `$\alpha$` for inline math expr or
```
$$
\alpha
$$
```
for blocked math expr.